tools/sgen/input/FluentUIStyle.yml

```diff
@@ -230,6 +230,15 @@ MSFOverflowAvatarTokens extends MSFAvatarTokens:
   foregroundDefaultColor: $Colors.Foreground.neutral3
   ringDefaultColor: $Colors.Background.neutralDisabled
 
+AP_MSFAvatarGroupTokens:
```

Comment:
It seems like the only token we need is the interspace.
Why do we need ringInnerGap, ringThickness and ringOuterGap?

Reply:
Thoughts on this one?

Reply:
It is used for the borders gap in a Stack

Reply:
Can you use the tokens defined for the Avatar instead of duplicating them for the AvatarGroup?